packages/solana/test/solana_client_test.dart

```diff
@@ -49,7 +49,7 @@ void main() {
     );
     expect(signature, isNotNull);
     expect(
-      await solanaClient.getLamports(address: destination.address),
+      await solanaClient.rpcClient.getBalance(destination.address),
```

Comment:
Can't we just create a `SolanaClient.getBalance()` method?

Reply:
Well, there's nothing special about this specific method, and if we'd go this way, we'll end up with proxying all the methods from `RpcClient` – I think it will be too much, having access to `rpcClient` seems to be simple enough.

Reply:
Yes, probably right. I agree.